HISTORY.rst

```diff
@@ -2,6 +2,55 @@
 History
```

Comment:
i'm going to try to fully fill out the history (and then ideally keep it up to date moving forwards)